services-geocoding/src/main/java/com/mapbox/api/geocoding/v6/models/V6Coordinates.java

```diff
@@ -0,0 +1,67 @@
+package com.mapbox.api.geocoding.v6.models;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.google.auto.value.AutoValue;
+import com.google.gson.Gson;
+import com.google.gson.TypeAdapter;
+import com.google.gson.annotations.SerializedName;
+import com.mapbox.geojson.Point;
+
+import java.io.Serializable;
+
+/**
+ * Object containing coordinate parameters (lat, long) and accuracy.
+ */
+@AutoValue
+public abstract class V6Coordinates implements Serializable {
+
+  /**
+   * Longitude of result.
+   *
+   * @return longitude of result
+   */
+  @NonNull
+  @SerializedName("longitude")
+  public abstract Double longitude();
+
+  /**
+   * Latitude of result.
+   *
+   * @return latitude of result
+   */
+  @NonNull
+  @SerializedName("latitude")
+  public abstract Double latitude();
+
+  /**
+   * Result coordinate as a {@code Point}.
+   *
+   * @return Result coordinate as a {@code Point}.
+   */
+  @NonNull
+  public Point point() {
+    return Point.fromLngLat(longitude(), latitude());
+  }
+
+  /**
+   * Point accuracy metric for the returned address feature.
+   *
+   * @return accuracy metric for the returned address feature
```

Comment:
What are the possible values?

Reply:
The list is here https://docs.mapbox.com/api/search/geocoding-v6/#point-accuracy-for-address-features. When API will be public I'll make `@StringDef` for this.